Pages/PostList.cshtml

```diff
@@ -1,35 +1,15 @@
-@page
-@model QCVault.Pages.PostListModel
+@page
+@model QCVault.Pages.PostListModel
 
 @section head
 {
 <meta property="og:title" content="The Motte Vault" />
 <meta property="og:description" content="The purpose of this community is to be a working discussion ground for people who may hold dramatically different beliefs. It is to be a place for people to examine the beliefs of others as well as their own beliefs; it is to be a place where strange or abnormal opinions and ideas can be generated and discussed fairly, with consideration and insight instead of kneejerk responses." />
 <meta property="og:image" content="https://picsum.photos/seed/frontpage/1200/630" />
 }
-
-<div class="post-list">
-    @foreach (var post in Model.Posts)
-    {
-    <div class="post-list-element">
-        <div class="thumbnail">
-            <a asp-page="Post" asp-route-title="@post.URLSlug">
-                @* This is just a temporary image placeholder until we have images *@
-                <img src=@($"https://picsum.photos/seed/{post.URLSlug}/160/100") width="160" height="100" loading="lazy">
-            </a>
-        </div>
-        <div class="title"><a asp-page="Post" asp-route-title="@post.URLSlug">@post.Title</a></div>
-        <div class="metadata">
-            <div class="author">@post.Author</div>
-            <div class="divider">-</div>
-            <div class="date">@post.Date.ToString("yyyy MMM d")</div>
-        </div>
-        <div class="tags">
-            @foreach (var category in post.Category)
-            {
-                <span class="tag tag-@category">@category</span>
-            }
-        </div>
-    </div>
-    }
-</div>
+
+<div class="post-list">
+	@foreach (var post in Model.Posts) {
+		@await Component.InvokeAsync("Post", post)

```

Comment:
Try to use the `<vc:name />` syntax, you can find an example of it by searching for `vc:share-widget`; I also just realized I missed an example of that, so I'll either fix it or let you finish this work first to avoid interfering.

Reply:
Solved  by [3d45b88370f6f0e2afe3ec0b534ee917ff3143ab](https://github.com/themotte/vault/pull/110/commits/3d45b88370f6f0e2afe3ec0b534ee917ff3143ab)